homeassistant/components/strings.config_entry_example.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I think that any component that will be 2 or more files should be converted to a folder based component.

```
hue/__init__.py
hue/strings.json
```

Because otherwise I think that our component folder will become a mess.